saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -184,8 +186,11 @@ setup_ccmv2() {
   chmod 400 "$TRUSTED_PROXY_CERT_PATH"
 
   INVERTING_PROXY_URL="$CCM_V2_INVERTING_PROXY_HOST"
+  
+  # A more sophisticated solution might need to be patched in later - tbh the script originally expected a full url with protocol scheme and closing slash
+  INVERTING_PROXY_FULL_URL="https://$INVERTING_PROXY_URL/"
```

Comment:
AFAIK this URL is coming from the java code - validating / extending the URL scheme should be done there, but for the time being and for this ticket, we can avoid the CB java code change.